curriculum/locales/english/learn-the-metaplex-sdk-by-minting-an-nft.md

```diff
@@ -1367,7 +1367,7 @@ const minifiedCode = babelisedCode.generateCode(variableDeclaration, {
 });
 assert.match(
   minifiedCode,
-  /imageBuffer=(await )?(readFile|readFileSync|read)\(('|"|`)(\.\/)?assets\/pic\.png\3\)/
+  /imageBuffer=(await readFile|readFileSync|read)\(('|"|`)(\.\/)?assets\/pic\.png\2\)/
```

Comment:
<s>We cannot force `await readFileSync`, though.</s>

I keep forgetting the order of operation for regex alternatives `|`.